src/ensembl/src/header/launchbar/Launchbar.scss

```diff
@@ -77,14 +77,31 @@
   }
 }
 
-.about {
+.logotype {
+  height: 12px;
+  fill: white;
+}
+
+.aboutEnsembl {
   display: flex;
   align-items: center;
-  margin-left: 20px;
+  padding-right: 46px;
+  font-size: 13px;
+  color: $blue;
+}
+
+.logotypeAbout {
```

Comment:
Perhaps if the `logotype` class isn't used anywhere in the Launchbar component, this class can be renamed to just .logotype?